packages/falso/src/tests/clothing-size.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { randClothingSize } from '../lib/clothing-size';
+
+describe('clothingSize', () => {
+  it('should create', () => {
+    expect(randClothingSize).toBeTruthy();
+  });
```

Comment:
You can remove this spec, if the function won't exist for some reason the other specs will fail.